pkgs/top-level/all-packages.nix

```diff
@@ -20994,7 +20994,7 @@ with pkgs;
 
   qgnomeplatform = libsForQt5.callPackage ../development/libraries/qgnomeplatform { };
 
-  randomx = callPackage ../development/libraries/randomx { };
+  randomx = darwin.apple_sdk_11_0.callPackage ../development/libraries/randomx { };
```

Comment:
Required to access `pthread_jit_write_protect_np`. See https://github.com/NixOS/nixpkgs/pull/186502#issuecomment-1214310441

Reply:
Not that it's your fault, but I don't really like this design: a particular platform shouldn't "take over" the package definition, also it seems it doesn't compose.

Reply:
By the way, your comment should be a comment in the source code.

Reply:
It's a hopefully temporary fix while the sdk issues are being resolved https://github.com/NixOS/nixpkgs/pull/176661 but yeah, for now we need to put up with some imprefect solutions.